testDirectory/1.ics

```diff
@@ -0,0 +1,12 @@
+BEGIN:VCALENDAR
```

Comment:
Seems that the content of this directory should be part of .gitignore
And it should have better name.

Reply:
Done!

Reply:
I added logic to delete them in
```
@AfterEavh
void cleanUp()
```